ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJBTransformers.java

```diff
@@ -301,4 +321,70 @@ private static void registerPassivationStoreTransformers_1_2_1_and_1_3_0(Resourc
                 .setValueConverter(AttributeConverter.Factory.createHardCoded(new ModelNode().set(TimeUnit.SECONDS.name()), true), EJB3SubsystemModel.IDLE_TIMEOUT_UNIT)
         ;
     }
+
+    /**
+     * This Combined Transformer manages this transformation from EAP7 to previous versions (attribute definition/xml markup):
+     *
+     * DEFAULT_SFSB_CACHE / cache-ref --> DEFAULT_CLUSTERED_SFSB_CACHE / clustered-cache-ref
+     * DEFAULT_SFSB_PASSIVATION_DISABLED_CACHE / passivation-disabled-cache-ref --> DEFAULT_SFSB_CACHE / cache-ref
+     *
+     */
+    private static class SfsbChacheRefTransformer implements CombinedTransformer {
```

Comment:
Can we fix the spelling?  "StatefulCacheRefTransformer" is probably clearer.

Reply:
@pferraro I've fixed the spelling problem.